robot/EDA/resources/EDA.py

```diff
@@ -329,9 +332,22 @@ def populate_placeholder(self, loc, value):
         field = self.selenium.get_webelement(xpath)
         field.send_keys(value)
         time.sleep(1)
-#         if loc == ("Search Contacts" or "Search Accounts"):
         field.send_keys(Keys.ARROW_DOWN + Keys.ENTER)
 
+    def format_all(self, loc, value):
+        """ Formats the given locator with the value for all {} occurrences """
+        count = 0
```

Comment:
Lines 339-342 can be replaced with:

```suggestion
        count = loc.count('{')
```